djconnectwise/sync.py

```diff
@@ -229,6 +229,16 @@ def _assign_field_data(self, company, company_json):
         company.zip = company_json.get('zip')
         company.created = timezone.now()
         company.deleted_flag = company_json.get('deletedFlag', False)
+
+        status_json = company_json.get('status')
+        
+        if status_json:
+            try:
+                status = models.CompanyStatus.objects.get(pk=status_json['id'])
+                company.status = status
+            except:
```

Comment:
Hi @AparatTechnologies . Please adjust this to catch something more specific. There must be a pattern elsewhere in `sync.py`.

I don't accept merge requests with `except:`, unless it's specifically for catching all errors and the handler includes logging that sends the stack trace somewhere.

Thanks.